libkbfs/bcache.go

```diff
@@ -67,15 +72,15 @@ func NewBlockCacheStandard(transientCapacity int,
 	return b
 }
 
-// Get implements the BlockCache interface for BlockCacheStandard.
-func (b *BlockCacheStandard) Get(ptr BlockPointer) (Block, error) {
+// GetWithPrefetch implements the BlockCache interface for BlockCacheStandard.
+func (b *BlockCacheStandard) GetWithPrefetch(ptr BlockPointer) (Block, bool, error) {
 	if b.cleanTransient != nil {
 		if tmp, ok := b.cleanTransient.Get(ptr.ID); ok {
-			block, ok := tmp.(Block)
+			bc, ok := tmp.(blockContainer)
 			if !ok {
-				return nil, BadDataError{ptr.ID}
+				return nil, false, BadDataError{ptr.ID}
 			}
-			return block, nil
+			return bc.block, bc.hasPrefetched, nil
```

Comment:
I'm about to add some code that will use the `false` version of `PutWithPrefetch`.